resources/scripts/components/server/backups/BackupContainer.tsx

```diff
@@ -60,7 +60,7 @@ const BackupContainer = () => {
             </Pagination>
             {backupLimit === 0 &&
             <p css={tw`text-center text-sm text-neutral-300`}>
-                Backups cannot be created for this server.
+                Backups cannot be created for this server when the backup limit is set to 0.
```

Comment:
Wouldn't it make more sense to do
"because the backup limit is set to 0."

Reply:
@TASelwyn how so? imo it's alot easier for people to understand.

Reply:
I also prefer the use of "because" rather than "when" in this context as it is more clear as the limit being 0 is the *cause* of not being able to create backups.

Reply:
> @TASelwyn how so? imo it's alot easier for people to understand.

Based on the context, "because" makes more grammarical sense and more clearly defines what is the cause.